coderd/notifications_test.go

```diff
@@ -60,24 +60,4 @@ func TestUpdateNotificationsSettings(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, expected, actual)
 	})
-
-	t.Run("Settings not modified", func(t *testing.T) {
```

Comment:
This seems like a useful test; curious why you removed it?

Reply:
It just tests that `PutNotificationsSettings` does not fail, so I wasn't convinced about its value. I can bring it back, not a big deal 👍